ckan/logic/schema.py

```diff
@@ -370,6 +377,16 @@ def default_extras_schema(ignore: Validator, not_empty: Validator,
     }
 
 
+@validator_args
+def default_show_extras_schema(ignore: Validator):
+
+    schema = default_extras_schema()
+
+    schema["id"] = [ignore]
```

Comment:
does this strip the ids out of the extras on show? If so I love it.

Reply:
Yes! But that has been the case for a really long time https://github.com/ckan/ckan/commit/5cd0401e13bad2c2a3a279b13d141fc524191ac4